src/macros-by-example.adoc

```diff
@@ -115,14 +115,14 @@ expression.  Here’s a silly macro to illustrate:
 
 [{mrk}]
 ----
-(*macro* reverse [a, c] [c, a])
+(*macro* reverse [a, b] [b, a])
 ----
 ----
 (:reverse first 1990) ⇒ [1990, first]
 ----
 
 The sub-expressions in these templates demonstrate that the expression language treats symbols as
-variable references.  Each `a` and `c` in the templates above refer to the parameters of the
+variable references.  Each `a`, `b`, and `c` in the templates above refer to the parameters of the
```

Comment:
Is the `c` here referring to the one declared on line 88?

```
(*macro* price
  [a, c]                          // signature
  { amount: a, currency: c })     // template
```

Reply:
Yeah, that was the intent, but it's awkward for sure.  Ideas to clarify?

Reply:
Maybe a rephrasing parenthetical?

> The symbols in the templates above (`a` and `c` in the `price` macro, `a` and `b` in the `reverse` macro) refer to the parameters of their respective surrounding macros [....]

Reply:
Rephrased in #226 thanks for the great suggestion!